controllers/app.controller.js

```diff
@@ -13,4 +14,11 @@ exports.getApi = (req, res) => {
     checkEndpoints().then((data) => {
         res.status(200).send(data);
     });
+}
+
+exports.getArticle = (req, res) => {
```

Comment:
this function should be more semantic. As you only want the article by the article id it should be something like getArticleById here.